### PR TITLE
Bugfix/single storeview bug

### PR DESCRIPTION
**Pull request by DanielRiezebos**
I noticed a bug when assigning a Wordpress page to a single store view. After a careful investigation I discovered that our PageHelper class was not really fit for this feature. After unsuccessfully wrestling with Magento 2's own UrlBuilderInterface I decided to build my own buildUrlFromPage() function to do the work for me.

Of course I have tested this on my environment without much hassle. 

I would like to know what you think!

### CODE REVIEW

File: Plugin/Model/SavePlugin.php [hunk 1]
```diff
@@ -79,4 +80,19 @@ public function afterSave(PageRepository $subject, Page $result)
...
     }
+
+    private function buildUrlFromPage(Page $page): string
+    {
+        if (!array_key_exists(0, $page->getStoreId())) {
+            return '';
+        }
+
+        try {
+            $storeFromPage = $this->storeManager->getStore($page->getStoreId()[0]);
```
Comment (tdgroot) [inline Plugin/Model/SavePlugin.php:91]: Just a suggestion:

```suggestion
            $storeIds = $page->getStoreId();
            if (!count($storeIds)) {
                return '';
            }
            
            try {
                $firstStoreId = $storeIds[0];
                $storeFromPage = $this->storeManager->getStore($firstStoreId);
                
                return $storeFromPage->getBaseUrl() . $page->getIdentifier();
            } catch (NoSuchEntityException $e) {
                return '';
            }
```

File: Plugin/Model/SavePlugin.php [hunk 2]
```diff
@@ -10,10 +10,11 @@
 use Mooore\WordpressIntegrationCms\Model\HttpClient\Page as PageClient;
```
Comment (tdgroot) [inline Plugin/Model/SavePlugin.php:10]: Please reorder imports

**Review by tdgroot — CHANGES_REQUESTED** [review]

File: Plugin/Model/SavePlugin.php (hunk 1 above)
Comment (DanielRiezebos) [reply Plugin/Model/SavePlugin.php:91]: Interesting suggestion! Why do you not explicitly check for index `0` to be in the output of `$page->getStoreId()`?

Comment (tdgroot) [reply Plugin/Model/SavePlugin.php:91]: Readability: it's not about specific key `0`, but rather about if the array is filled.

File: Plugin/Model/SavePlugin.php [hunk 3]
```diff
@@ -79,4 +80,31 @@ public function afterSave(PageRepository $subject, Page $result)
 
         return $result;
     }
+
+    private function buildUrlFromPage(Page $page): string
+    {
+        if (!array_key_exists(0, $page->getStoreId())) {
```
Comment (tdgroot) [inline Plugin/Model/SavePlugin.php:86]: Is this check still necessary since line 92?

Comment (DanielRiezebos) [reply Plugin/Model/SavePlugin.php]: Whoops! That was messy, my bad. I've fixed it in a new commit.

**Review by tdgroot — APPROVED** [review]

**Pull request merged**